modules/installation-configuration-parameters.adoc

```diff
@@ -1544,23 +1557,23 @@ Additional Alibaba Cloud configuration parameters are described in the following
 |String list.
 
 |`platform.alibabacloud.defaultMachinePlatform.imageID`
-|For compute machines, the image ID that should be used to create ECS instance. If set, the image ID should belong to the same region as the cluster
+|For both compute machines and control plane machines, the image ID that should be used to create ECS instance. If set, the image ID should belong to the same region as the cluster.
 |String.
 
 |`platform.alibabacloud.defaultMachinePlatform.instanceType`
-|For compute machines, the configuration used when installing on Alibaba Cloud.
+|For both compute machines and control plane machines, the configuration used when installing on Alibaba Cloud.
```

Comment:
The statement doesn't tell the exact meaning of the parameter, suggest to replace `the configuration used when installing on Alibaba Cloud` with `the ECS instance type that should be used to create ECS instance`.